release step failed at step  monkeyRelease

Looks like caused by gradle update.

```
21:23:06.983 [DEBUG] [org.gradle.internal.progress.DefaultBuildOperationExecutor] Build operation 'Task :encryptionExample:installDeviceRelease' completed
21:23:06.983 [INFO] [org.gradle.execution.taskgraph.DefaultTaskPlanExecutor] :encryptionExample:installDeviceRelease (Thread[Task worker for ':' Thread 10,5,main]) completed. Took 0.35 secs.
21:23:06.984 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
21:23:06.984 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
21:23:06.985 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
21:23:06.985 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Run with --stacktrace option to get the stack trace. 
21:23:06.985 [ERROR] [org.gradle.internal.buildevents.BuildResultLogger] BUILD FAILED in 1s
21:23:06.986 [DEBUG] [org.jetbrains.kotlin.gradle.plugin.KotlinGradleBuildServices] [KOTLIN] Disposed KotlinGradleBuildServices
21:23:06.987 [INFO] [org.gradle.cache.internal.DefaultPersistentDirectoryCache] Build cache (/home/cc/.gradle/caches/build-cache-1) has not been cleaned up in 2 days
21:23:06.987 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Daemon worker Thread 7: released lock on root.1
21:23:06.987 [DEBUG] [org.gradle.internal.progress.DefaultBuildOperationExecutor] Completing Build operation 'Run build'
21:23:06.988 [DEBUG] [org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess] Releasing file lock for file content cache (/home/cc/repo/realm/release/examples/.gradle/4.1/fileContent)
21:23:06.988 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on file content cache (/home/cc/repo/realm/release/examples/.gradle/4.1/fileContent).
21:23:06.989 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.resolveengine.store.CachedStoreFactory] Resolution result cache closed. Cache reads: 0, disk reads: 0 (avg: 0.0 secs, total: 0.0 secs)
Creating configuration api
Creating configuration implementation
Creating configuration testApi
Creating configuration testWearApp
Creating configuration testDebugCompile
Appending classLoaderHash to build cache key: 40ca7913578478862f2e0efd4d30b6bd
Appending actionType to build cache key: org.gradle.api.internal.project.taskfactory.AbstractOutputPropertyAnnotationHandler$2$1
:encryptionExample:checkReleaseManifest (Thread[Task worker for ':',5,main]) started.
[ 69%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 70%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 71%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 72%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 73%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 75%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 76%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 77%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 78%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 80%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 81%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 82%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 83%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 85%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 86%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 87%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 88%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 89%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 91%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 92%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 93%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 94%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 96%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 97%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 98%] /data/local/tmp/encryptionExample-release-unsigned.apk
[ 99%] /data/local/tmp/encryptionExample-release-unsigned.apk
[100%] /data/local/tmp/encryptionExample-release-unsigned.apk
/home/cc/repo/realm/release/examples/encryptionExample/build/outputs/apk/release/encryptionExample-release-unsigned.apk: 1 file pushed. 99.6 MB/s (5317983 bytes in 0.051s)
        pkg: /data/local/tmp/encryptionExample-release-unsigned.apk
Failure [INSTALL_PARSE_FAILED_NO_CERTIFICATES]


:encryptionExample:installDeviceRelease (Thread[Task worker for ':',5,main]) completed. Took 0.346 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':encryptionExample:installDeviceRelease'.
> Installation failed with error: INSTALL_PARSE_FAILED_NO_CERTIFICATES

* Try:
Run with --stacktrace option to get the stack trace. Run with --debug option to get more log output.

```

--- a/examples/encryptionExample/build.gradle
+++ b/examples/encryptionExample/build.gradle
@@ -16,7 +16,11 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
     productFlavors {

--- a/examples/gridViewExample/build.gradle
+++ b/examples/gridViewExample/build.gradle
@@ -15,7 +15,11 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
     productFlavors {

--- a/examples/introExample/build.gradle
+++ b/examples/introExample/build.gradle
@@ -16,7 +16,11 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
 

--- a/examples/jsonExample/build.gradle
+++ b/examples/jsonExample/build.gradle
@@ -16,7 +16,11 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
     productFlavors {

--- a/examples/kotlinExample/build.gradle
+++ b/examples/kotlinExample/build.gradle
@@ -32,7 +32,11 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
 

--- a/examples/migrationExample/build.gradle
+++ b/examples/migrationExample/build.gradle
@@ -15,7 +15,11 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
     command {

--- a/examples/newsreaderExample/build.gradle
+++ b/examples/newsreaderExample/build.gradle
@@ -15,7 +15,9 @@ android {
     }
     buildTypes {
         release {
+            // FIXME: Fix the proguard with 3rd party libs
             minifyEnabled false
+            signingConfig signingConfigs.debug
         }
     }
     command {

--- a/examples/objectServerExample/build.gradle
+++ b/examples/objectServerExample/build.gradle
@@ -43,10 +43,12 @@ android {
         def host = getIP()
         debug {
             buildConfigField "String", "OBJECT_SERVER_IP", "\"${host}\""
+            minifyEnabled true
         }
         release {
-            minifyEnabled false
             buildConfigField "String", "OBJECT_SERVER_IP", "\"${host}\""
+            minifyEnabled true
+            signingConfig signingConfigs.debug
         }
     }
 

--- a/examples/rxJavaExample/build.gradle
+++ b/examples/rxJavaExample/build.gradle
@@ -19,7 +19,9 @@ android {
 
     buildTypes {
         release {
+            // FIXME: Fix the proguard with 3rd party libs
             minifyEnabled false
+            signingConfig signingConfigs.debug
         }
     }
 

--- a/examples/secureTokenAndroidKeyStore/build.gradle
+++ b/examples/secureTokenAndroidKeyStore/build.gradle
@@ -18,7 +18,12 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+        debug {
+            minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }

--- a/examples/threadExample/build.gradle
+++ b/examples/threadExample/build.gradle
@@ -15,7 +15,11 @@ android {
     }
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
     command {

--- a/examples/threadExample/src/main/java/io/realm/examples/threads/ThreadFragment.java
+++ b/examples/threadExample/src/main/java/io/realm/examples/threads/ThreadFragment.java
@@ -169,6 +169,7 @@ public class ThreadFragment extends Fragment {
     public void onStop() {
         super.onStop();
         // Remember to close the Realm instance when done with it.
+        dotsView.setRealmResults(null);
         realm.close();
     }
 }

--- a/examples/threadExample/src/main/java/io/realm/examples/threads/widget/DotsView.java
+++ b/examples/threadExample/src/main/java/io/realm/examples/threads/widget/DotsView.java
@@ -77,6 +77,9 @@ public class DotsView extends View {
     @Override
     protected void onDraw(Canvas canvas) {
         super.onDraw(canvas);
+        if (results == null) {
+            return;
+        }
         canvas.drawColor(Color.TRANSPARENT);
         for (Dot dot : results) {
             circlePaint.setColor(dot.getColor());

--- a/examples/unitTestExample/build.gradle
+++ b/examples/unitTestExample/build.gradle
@@ -19,7 +19,11 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            signingConfig signingConfigs.debug
+        }
+        debug {
+            minifyEnabled true
         }
     }
 

